data/styles/clear/style-clear/symbols/aircraft-m.svg

```diff
@@ -0,0 +1,4 @@
+<svg version="1.1" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Add `<?xml version="1.0" encoding="UTF-8"?>` header in a new first line in all svg files

Reply:
done 👍